docker/fedora-41-nightly.Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM fedora:41
 
+# Bust the cache
+ARG STAMP=1741531139
```

Comment:
Completely unrelated, but surprised this is needed for GH actions; I always pictured this as a workaround for CirrusCI not correctly detecting changes to the used images (probably fixed since ages, also possible to work around by explicitly running the Cirrus image build task when it was skipped).

Reply:
Yeah, don't think GH needs it, but it helps busting the local cache for the nightly target, so now it'll just pull all the dependencies afresh every day when running `make check-nightly` instead of just the Zeek packages.

https://github.com/corelight/zeekjs/blob/79b0c2126fa0178dbc2e37536588fcd1db9f4443/Makefile#L47-L48